### ISSUE TITLE
Option to not run build runner after generating JSON to Dart

### ISSUE THREAD

**Issue by 2shrestha22** [hiranthaR/Json-to-Dart-Model#40]
This extension is awesome and thanks for developing this. I don't like to run build runner instantly for a single time. I prefer to run build runner with watch mode which I am manually doing while working with freezed. So it will be great to have a option to disable build runner after generating JSON to Dart.

**Comment by iamarnas** [comment]
@2shrestha22 Hi 👋
Thank you for your suggestion. It is easy to fix. What do you think maybe it's better allow choose run or watch builder in the settings instead to disable it. Or disabling it is the best option. I do not work with Freezed it therefore I ask you what do you think about it.

**Comment by 2shrestha22** [comment]
@iamarnas Currently when new freezed data class is created builder runner, opens a new terminal and builds. So I think completely disabling build runner will be a good option because everyone using freezed must have known build runner too. 😀

**Comment by iamarnas** [comment]
> @iamarnas Currently when new freezed data class is created builder runner, opens a new terminal and builds. So I think completely disabling build runner will be a good option because everyone using freezed must have known build runner too. 😀

@2shrestha22 OK. Right now I'm on vacation. I fix it when I find free time.

**Comment by 2shrestha22** [comment]
@iamarnas ok thank you

**Comment by iamarnas** [comment]
@2shrestha22 Hi again!

I added option to disable build runner in the `Settings/Extensions/JSON To Dart Model`. Let me know if it works as you expected. Thank you.

**Comment by 2shrestha22** [comment]
Which option should we use to disable build runner? I have 4 but it is still running build runner in terminal.
![image](https://user-images.githubusercontent.com/24835256/125240019-3fd3fb80-e309-11eb-9000-8881279a263d.png)

**Comment by iamarnas** [comment]
@2shrestha22 Hi 👋 
> Which option should we use to disable build runner? I have 4 but it is still running build runner in terminal.

 Have you latest version 3.2.5?

**Comment by 2shrestha22** [comment]
Yes I have 3.2.5

**Comment by iamarnas** [comment]
@2shrestha22 Then you would see the new option `Run Builder`.

![builder](https://user-images.githubusercontent.com/42679871/125256301-9dae1680-e2fc-11eb-9445-c1e57043ffa2.png)

**Comment by 2shrestha22** [comment]
@iamarnas Let me try again. And I will inform you.

**Comment by 2shrestha22** [comment]
Its working now, found new setting as you described. I will suggest a new feature. What about option in Explorer to create model class there? 
![Peek 2021-07-12 17-16](https://user-images.githubusercontent.com/24835256/125280535-dff34a00-e334-11eb-8370-2c6d5dbdd302.gif)

**Comment by iamarnas** [comment]
@2shrestha22 Thanks for suggestion. 

> I will suggest a new feature. What about option in Explorer to create model class there?

This feature should work same as from the clipboard or from the file models just would pick faster target directory. If you setup JSON Dart To Model configuration in the settings and run fast mode. You can generate JSON string in few seconds by using hot keys. Tips: Try hot keys :)
